src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -1398,7 +1398,7 @@ public void Process_StartTest()
             string domain = "thisDomain";
             SecureString password = AsSecureString("Value");
 
-            Process p = Process.Start(currentProcessName, userName, password, domain);
+            Process p = Process.Start(currentProcessName, userName, password, domain); // This writes junk to the Console but with this overload, we can't prevent that.
```

Comment:
Yes: this overload does not allow redirecting the output, obviously. We could invoke something else. I don't have an issue for that.